Makefile.am

```diff
@@ -37,6 +37,10 @@ MAINTAINERCLEANFILES	= Makefile.in aclocal.m4 configure depcomp \
 
 ACLOCAL_AMFLAGS		= -I m4
 
+DISTCHECK_CONFIGURE_FLAGS = --with-socket-dir=/tmp/libqb-$(VERSION)-$(shell echo $$$$)
```

Comment:
Only to be removed with subsequent commit?

Reply:
Yes, I decided it was better done in the CI itself rather than hard-coding it in the Makefile.

Reply:
But ... introducing temporary, unjustified inter-patchset noise is not
nice should anyone care about history, bisecting etc.

Reply:
It would have disappeared when I committed the final patch.

Reply:
I thought the typical approach is to evolve the patchset towards the final
form directly as a part of the review.  I see you want to see the response
from the CI, but cannot you prototype using your fork for that?

Reply:
TBH I just want something that works. I hate working with autotools and it's ghastly friends. Every keypress I make regarding them makes me less and less motivated.